lib/ferrum/page/screenshot.rb

```diff
@@ -61,6 +64,28 @@ def save_file(path, data)
         File.open(path.to_s, "wb") { |f| f.write(bin) }
       end
 
+      def stream_to_file(stream_handle, path)
+        File.open(path, 'wb') do |output_file|
+          stream_to stream_handle, output_file
+        end
+      end
+
+      def stream_to_memory(stream_handle)
+        in_memory_data = ''
+        stream_to stream_handle, in_memory_data
+        in_memory_data
+      end
+
+      def stream_to(stream_handle, output)
+        loop do
+          read_result = command("IO.read", handle: stream_handle, size: 131072)
```

Comment:
@nenoganchev would a constant be helpful here? `CHUNK_SIZE = 128 * 1024`

Please excuse the drive-by review – I was intrigued by this PR. The memory saving a great result 🎉

Reply:
@odlp: Sure, `131072` won't win any readability awards :) A constant defined right above this loop would help.